pos_lot_selection/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=================
+POS Lot Selection
+=================
+
+This module allows to pick betwenn existing lots in POS frontend.
+
+Usage
```

Comment:
Add `Configuration` section for talking about:

* Enable the option for managing lots.
* Configure in a product to use lots.